tests/framework/ROM/tensorflow_keras/keras_lstm_regression.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" ?>
+<Simulation>
+  <TestInfo>
+    <name>framework/ROM/tensorflow_keras/keras_lstm_regression</name>
+    <author>@cogljj</author>
+    <created>2021-04-13</created>
+    <revisions>
+    </revisions>
+    <classesTested>KerasLSTMRegression</classesTested>
+    <description>
+      Test the Keras LSTM Regression.
```

Comment:
Add more description, for example, what kind of model are you using

Reply:
Added more detail.